spyder_kernels/ipdb/kernel.py

```diff
@@ -162,6 +162,71 @@ def do_is_complete(self, code):
             r['indent'] = ' ' * indent_spaces
         return r
 
+    def do_complete(self, code, cursor_pos):
```

Comment:
Did you copy this method from MetaKernel? If that's the case, what if we submit a PR to them instead to disable their `path_matches` and only use our own?

I think that should simpler than this.

Reply:
Yep, I took the same method although what I did was to do a replacement of the `"` for each element in `info['path_matches']` to prevent duplicated entries when extending the matches with the ones returned by `get_completions` (validating if the `get_completions` matches are the same as the ones that Metakernel retrieves).

Should I do a PR against Metakernel then?

Reply:
Yes, I think it's better. If we can't use `super(Metakernel, self).do_complete(...)` at some point in this re-implementation, then that's the way to go to maintain our own implementation of `do_complete`.